pyscf/adc/dfadc.py

```diff
@@ -15,6 +15,47 @@
 
 import numpy as np
 
+def  get_ovvv_df(myadc, Lov, Lvv, p, chnk_size):
+
+    nocc = myadc._nocc
+    nvir = myadc._nvir
+    naux = myadc.with_df.get_naoaux()
+
+    Lvv = Lvv.reshape(naux,nvir*nvir)
+    Lov = Lov.reshape(naux,nocc,nvir)
+
+    if chnk_size < nocc:
+        Lov_temp = np.ascontiguousarray(Lov.transpose(1,2,0)[p:p+chnk_size].reshape(-1,naux))
+    else :
+        Lov_temp = np.ascontiguousarray(Lov.transpose(1,2,0).reshape(-1,naux))
+
+    ovvv = np.dot(Lov_temp, Lvv)
+    ovvv = ovvv.reshape(-1, nvir, nvir, nvir)
+    return ovvv    
+
+
+def  get_ovvv_a_df(myadc, Lov, Lvv, p, chnk_size):
+
```

Comment:
Docstring missing here, too: how is this different from `get_ovvv_df`?

Reply:
`get_ovvv_df` is used in the restricted code, whereas `get_ovvv_a_df` is used in the unrestricted code to compute different spin cases of ovvv. I have added docstrings and renamed `get_ovvv_a_df` to `get_ovvv_spin_df`.